apps/web-client/src/lib/components/Tables/InventoryTables/OutboundTable.svelte

```diff
@@ -54,7 +55,18 @@
 	<tbody>
 		{#each rows as row (row.key)}
 			{#if isBookRow(row)}
-				{@const { rowIx, isbn, authors = "N/A", quantity, price, year = "N/A", title = "N/A", publisher = "", warehouseDiscount } = row}
+				{@const {
+					rowIx,
+					isbn,
+					authors = "N/A",
+					quantity,
+					price,
+					year = "N/A",
+					title = "N/A",
+					publisher = "",
+					warehouseDiscount,
+					category = ""
```

Comment:
The custom item row is out of sync with the dynamic table. We need to add an additional `<td>` there to account for Category col space on big screen. 

Additionally there's a bug from before: these empty `td`s should use the same responsive classes as a normal row.